.env.example

```diff
@@ -4,4 +4,5 @@ LISTEN_GROUP=signal groupId for the group to listen to
 TWITTER_API_KEY=Twitter API key
 TWITTER_API_SECRET=Twitter API secret
 TWITTER_ACCESS_TOKEN=Twitter access token
-TWITTER_TOKEN_SECRET=Twitter access token secret
\ No newline at end of file
+TWITTER_TOKEN_SECRET=Twitter access token secret
+TIMEZONE=tz_database format timezone (see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones for complete list, default is UTC)
```

Comment:
I think that this env var may not be needed any more since no conversions are being done

Reply:
Whoops, correct, cleaning these up.